packages/static/kairos-overlay-files/files/etc/cos/bootargs.cfg

```diff
@@ -1,7 +1,8 @@
 function setSelinux {
     source (loop0)/etc/os-release
     set baseSelinuxCmd=""
-    if regexp "rockylinux|fedora|almalinux|redhat|tumbleweed" $KAIROS_ARTIFACT; then
+    # opensuse does not bundle regexp module so if we are on leap or tumbleweed we need to do this manually
+    if test $KAIROS_FLAVOR == "rockylinux" -o test $KAIROS_FLAVOR == "fedora" -o test $KAIROS_FLAVOR == "almalinux" -o test $KAIROS_FLAVOR == "redhat" -o test $ID == "opensuse-tumbleweed"; then
```

Comment:
We can stick to `KAIROS_` prefixed variables, which we fully  control, if you use `$KAIROS_FLAVOR_RELEASE == "tumbleweed"` instead of `ID`.

Reply:
but do we have that variable already or you think we should introduce it? I thin what its there should be enough to identify, especially this as this is tumbleeed only

Reply:
It's there: https://github.com/kairos-io/kairos-sdk/blob/c21aa7f149cbbea5416f7fd475d9c6613d208d4e/versioneer/versioneer.go#L18 . But it's ok as it is too. It's minor.